test/e2e/petset.go

```diff
@@ -581,14 +721,22 @@ func (p *petSetTester) setHealthy(ps *apps.PetSet) {
 			framework.Failf("Found multiple non-healthy pets: %v and %v", pod.Name, markedHealthyPod)
 		}
 		p, err := framework.UpdatePodWithRetries(p.c, pod.Namespace, pod.Name, func(up *api.Pod) {
-			up.Annotations[petset.PetSetInitAnnotation] = "true"
+			up.Annotations[petset.PetSetInitAnnotation] = value
 		})
 		ExpectNoError(err)
 		framework.Logf("Set annotation %v to %v on pod %v", petset.PetSetInitAnnotation, p.Annotations[petset.PetSetInitAnnotation], pod.Name)
 		markedHealthyPod = pod.Name
 	}
 }
 
+func (p *petSetTester) setHealthy(ps *apps.PetSet) {
```

Comment:
by done i meant that it was clear from the first comment, i changed it in my branch and will push as soon as it will be ready